@xen-orchestra/lite/src/components/pool/system/PoolNetworking.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <UiCard>
+    <UiTitle>
+      {{ $t('networking') }}
+    </UiTitle>
+    <VtsLoadingHero v-if="!isReady" type="card" :title="$t('networking')" />
+    <template v-else>
+      <VtsQuickInfoRow :label="$t('backup-network')">
+        <template #value>
+          <UiLink
+            v-if="pool.other_config['xo:backupNetwork'] && backupNetwork?.name_label"
+            :to="`/pool/${pool.uuid}/network?id=${pool.other_config['xo:backupNetwork']}`"
+            :icon="faNetworkWired"
+            size="medium"
+          >
+            {{ backupNetwork.name_label }}
+          </UiLink>
```

Comment:
```suggestion
<UiLink
            v-if="backupNetwork && backupNetwork?.name_label"
            :to="`/pool/${pool.uuid}/network?id=${backupNetwork?.uuid}`"
            :icon="faNetworkWired"
            size="medium"
          >
            {{ backupNetwork.name_label }}
          </UiLink>
```

I think we can do something like that, maybe do the same changes on xo 6 ?

Reply:
It can be simplified to:
```suggestion
          <UiLink
            v-if="backupNetwork !== undefined"
            :to="`/pool/${pool.uuid}/network?id=${backupNetwork.uuid}`"
            :icon="faNetworkWired"
            size="medium"
          >
            {{ backupNetwork.name_label }}
          </UiLink>
```